src/MovieList/MovieList.js

```diff
@@ -1,9 +1,18 @@
-import React from 'react'
+import React from 'react';
 import Movie from '../Movie/Movie';
+import { Redirect } from 'react-router-dom';
 import './MovieList.css';
 
 const MovieList = ({ movies }) => {
-  const movieTiles = movies.map(movie => {
+  if (!movies) {
+    return (
+      <>
+      <Redirect to='/' />
+      <h1>Refreshing the main page...</h1>
+      </>
```

Comment:
I think we should add another conditional statement here to display a header informing the user if no titles match their search input.